tests/test-cases/readers/cube-config/v1.3/data.csv

```diff
@@ -0,0 +1,3 @@
+Dim-1,Dim-2,Attr-1,Amount,Measure,Units
+2009,import,goods,100,Billions,Pounds
```

Comment:
Billions is never a measure, that's part of the unit. The Measure could be something like `Monetary Value`. Can you correct it so we're not misleading people with our test cases?